homeassistant/components/roon/translations/en.json

```diff
@@ -0,0 +1,27 @@
+{
+    "config": {
+        "abort": {
+            "already_configured": "[%key:common::config_flow::abort::already_configured_device%]"
+        },
+        "error": {
+            "cannot_connect": "[%key:common::config_flow::error::cannot_connect%]",
+            "duplicate_entry": "That entity has already been added.",
```

Comment:
Make sure to run the translations script in develop mode and commit the changes. This will sync the English default translation with strings.json.
```
python -m script.translations develop
```